commit_check/config.py

```
@@ -0,0 +1,32 @@
+def load_config(path_hint: str = "") -> Dict[str, Any]:
+    """Load and validate config from TOML file."""
+    if path_hint:
+        p = Path(path_hint)
+        if p.exists():
+            with open(p, "rb") as f:
+                return toml_load(f)
+    for candidate in DEFAULT_CONFIG_PATHS:
+        if candidate.exists():
+            with open(candidate, "rb") as f:
+                return toml_load(f)
+    raise FileNotFoundError("No config file found (cchk.toml or commit-check.toml)")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)